### PR TITLE
Add Pebble custom notice and change updated

### PR DESCRIPTION
**Pull request by tonyandrewmeyer**
Add the new Pebble events (custom notice and change updated) to the diagrams.

A few drive-by "juju" -> "Juju" as well :smile:.

TODO:
* [ ] Finish updating the simulator as well.
* [ ] Once the Juju docs have been updated make sure that the change update link is correct (currently it's a guess).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,7 +233,7 @@ sequenceDiagram
 
 ## Appendix 2: deferring an event
 
-This is the 'normal' way of using `defer()`: an event `event1` comes in but we are not ready to process it; we `defer()` it; when `event2` comes in, the OF will first flush the queue and fire `event1`, then fire `event2`. The ordering is preserved: `event1` is consumed before `event2` by the charm.
+This is the 'normal' way of using `defer()`: an event `event1` comes in but we are not ready to process it; we `defer()` it; when `event2` comes in, the oeprator framework will first flush the queue and fire `event1`, then fire `event2`. The ordering is preserved: `event1` is consumed before `event2` by the charm.
```
Comment (PietroPasotti) [inline README.md]:
```suggestion
This is the 'normal' way of using `defer()`: an event `event1` comes in but we are not ready to process it; we `defer()` it; when `event2` comes in, the operator framework will first flush the queue and fire `event1`, then fire `event2`. The ordering is preserved: `event1` is consumed before `event2` by the charm.
```

**Review by PietroPasotti — CHANGES_REQUESTED** [review]
Hi! Thanks for this one, unfortunately I saw this a few moments after I merged a commit where I added pebble custom notices and merged the two graphs into one.

Also worth noting that the single source of truth **for the text** is the discourse post: https://discourse.charmhub.io/t/charm-lifecycle/5938/16

so if you want to fix any typos do that there directly, otherwise we'll have to keep in sync manually.

We should probably delete everything but the graph(s) and the simulator from this repo to avoid confusion.

**Comment by tonyandrewmeyer** [comment]
> Hi! Thanks for this one, unfortunately I saw this a few moments after I merged a commit where I added pebble custom notices and merged the two graphs into one.

All good :smile:. Is it worth finishing up updating simulator for the two new Pebble types, or should I just close this?

**Comment by PietroPasotti** [comment]
the simulator is not being used and I wouldn't bother maintaining it at this stage, unless we want to look more into fuzz testing

**Comment by tonyandrewmeyer** [comment]
> the simulator is not being used and I wouldn't bother maintaining it at this stage, unless we want to look more into fuzz testing

I would actually, but I don't think I realistically have time for that at the moment.

**Pull request closed without merging**